docs/guide/workflow.md

```diff
@@ -150,7 +150,7 @@
 
     !!! note
 
-        This example only works if your workflow is a [direct acycling graph](https://en.wikipedia.org/wiki/Directed_acyclic_graph)
+        This example only works if your workflow is a [directed acyclic graph](https://en.wikipedia.org/wiki/Directed_acyclic_graph)
```

Comment:
```suggestion
        This example only works if your workflow is a [directed graph](https://en.wikipedia.org/wiki/Directed_graph).
```

Reply:
Hm,
 "_This example only works if your workflow is a [directed graph]_"
seems to be unnecessesary / redundant in a way - as workflows are supposed to be (or represented by) directed graphs in general.

Reply:
@reithor, so do you think it would be better to remove the note altogether? Will it be obvious to the reader?

Reply:
@DominikaK : ... missed your message, sorry.

'_This example only works if your workflow is a [direct acycling graph]_'
--> has wrong typo and is wrong
'_This example only works if your workflow is a [directed acyclic graph]_'
--> is wrong
'_This example only works if your workflow is a [directed graph]_'
--> gives no additional information

So yes, I would remove the note.